assets/app/scripts/services/data.js

```diff
@@ -108,10 +92,6 @@ angular.module('openshiftConsole')
     $rootScope.$on( "$routeChangeStart", function(event, next, current) {
       self._websocketEventsMap = {};
     });
-
-    this.oApiVersion = "v1";
-    this.k8sApiVersion = "v1";
```

Comment:
sadly, the brand new about page was using these. I'm going to remove the API versions from the about page until we have discovery in place and can actually show what APIs are available on the server.